specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2018-10-01/imds.json

```diff
@@ -0,0 +1,760 @@
+            "type": "object",
+            "description": "This is the response from the Identity_GetToken operation.",
+            "properties": {
+                "access_token": {
+                    "type": "string",
+                    "description": "This is the requested access token. The app can use this token to authenticate to the sink resource."
+                },
+                "expires_in": {
+                    "type": "string",
+                    "description": "This is how long the access token is valid (in seconds)."
+                },
+                "expires_on": {
+                    "type": "string",
+                    "description": "This is the time when the access token expires. The date is represented as the number of seconds from 1970-01-01T0:0:0Z UTC until the expiration time. This value is used to determine the lifetime of cached tokens."
+                },
+                "ext_expires_in": {
+                    "type": "string",
+                    "description": "This indicates the extended lifetime of the token."
+                },
+                "not_before": {
+                    "type": "string",
+                    "description": "This is the time when the access token becomes effective. The date is represented as the number of seconds from 1970-01-01T0:0:0Z UTC until the expiration time."
+                },
+                "resource": {
+                    "type": "string",
+                    "description": "This is the app ID URI of the sink resource."
+                },
+                "token_type": {
+                    "type": "string",
+                    "description": "This indicates the token type value."
+                },
+                "client_id": {
+                    "type": "string",
+                    "description": "This is the client_id specified in the request, if any."
+                },
+                "object_id": {
+                    "type": "string",
+                    "description": "This is the object_id specified in the request, if any."
+                },
+                "msi_res_id": {
+                    "type": "string",
+                    "description": "This is the msi_res_id specified in the request, if any."
+                }
+            }
+        },
+        "IdentityInfoResponse": {
+            "type": "object",
+            "description": "This is the response from the Identity_GetInfo operation.",
+            "properties": {
+                "tenantId": {
+                    "type": "string",
+                    "description": "This is the AAD tenantId of the identity of the caller."
+                }
+            }           
+        }
+    },
+    "parameters": {
+        "ApiVersionParameter": {
+            "name": "api-version",
+            "in": "query",
+            "type": "string",
+            "enum": [
+                "2018-02-01",
+                "2018-04-02",
+                "2018-10-01"
```

Comment:
We have one swagger per api version. What confuses me is that, you have the same apis for the three api versions.

If nothing changed between those api versions then why was the api version bumped?


FYI,
- The api version in the folder path and that in the top level info.version should be same.
- The api version in the top level info.version is used in sdk generation (client.apiVersion). It is a constant property and cannot be changed.